addons/web_editor/static/src/components/media_dialog/document_selector.js

```diff
@@ -49,7 +49,7 @@ export class DocumentSelector extends FileSelector {
     static async createElements(selectedMedia, { orm }) {
         return Promise.all(selectedMedia.map(async attachment => {
             const linkEl = document.createElement('a');
-            let href = `/web/content/${attachment.id}?unique=${attachment.checksum}&download=true`;
```

Comment:
I don't see this commit adding the option to download... was it already there and I don't remember it? :thinking:

Reply:
This commit is a minimal JS fix, we'd have to modify the link tools views to add that option, which is developed here in a more complete feature: https://github.com/odoo/odoo/pull/76313

Reply:
@younn-o You marked this PR ready for review 1 hour ago but did not reply nor change here? :thinking:

Reply:
Oups, did not receive the notification on that one... 
![image](https://user-images.githubusercontent.com/58561718/194298826-e47b1eab-8cc0-454a-ba26-da9ef24a2adf.png)
![image](https://user-images.githubusercontent.com/58561718/194299074-d7fe6a4f-3522-4efb-b03f-1e5c2ca98663.png)

For now, the behaviour is the following:
- A document viewable in the browser will be opened in the browser (in a new tab or not)
=> The user can still download it from the browser window  
- If it is not viewable in the browser, it will be downloaded automatically
When discussing with @pierre-pde, it was agreed that this behaviour was making sense.

But it is also possible to keep the `download=true`, say that all documents added from the media dialog are downloaded by default, and hide the "open in a new window" option from the link tools in that case.

@qsm-odoo @pierre-pde What do you think about it ?

Reply:
Ok, that's fine. I'll test and merge :+1:

Reply:
> After testing on Firefox, there seems to be a problem with the LinkPopover. I'll mark the PR as ready for review once solved
@qsm-odoo Unrelated to the PR, the problem is there in 16.0. I have created a task for that bug